HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+Utilize the result of eye-in-hand calibration to transform (picking) point
+coordinates from the camera frame to the robot base frame.
+*/
+
+#include <Eigen/Core>
+#include <cmath>
+
+#include <iostream>
+#include <opencv2/core/core.hpp>
+
+Eigen::MatrixXd cv2eigen(const cv::Mat &cv_mat)
+{
+    if(cv_mat.dims > 2)
+    {
+        throw "Invalid matrix dimensions. Expected 2D.";
+    }
+
+    Eigen::MatrixXd eigenMat(cv_mat.rows, cv_mat.cols);
+
+    for(int i = 0; i < cv_mat.rows; i++)
+    {
+        for(int j = 0; j < cv_mat.cols; j++)
+        {
+            eigenMat(i, j) = cv_mat.at<double>(i, j);
+        }
+    }
+
+    return eigenMat;
+}
+
+cv::Mat readTransform(const std::string file_name)
+{
+    auto fileStorage = cv::FileStorage::FileStorage();
+
+    if(!fileStorage.open(file_name, cv::FileStorage::Mode::READ))
+    {
+        std::cout << "Could not open file" << std::endl;
```

Comment:
If we can't open the file, we should not continue execution. I suggest throwing an exception here.

Reply:
done